source/includes/_documentation.md

```diff
@@ -4853,6 +4853,185 @@ The angular Button:
 **is()** | Assert action | TextAssert 
 
 #### <a href="https://github.com/jdi-testing/jdi-light/blob/master/jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BasicButtonTests.java" target="_blank">Basic Button java tests examples</a>
+
+### Button toggle
+
+```java     
+    //@FindBy(id = "underline-button-toggle-button") public static Button basicButtonToggleUnderline;
+    @UI("#underline-button-toggle-button")
+    public static Button basicButtonToggleUnderline;
+
+    @Test
+    public void basicButtonToggleTest() {
+        basicButtonToggleUnderline.is().enabled();
+        basicButtonToggleUnderline.is().displayed();
+        basicButtonToggleUnderline.has().text("Underline");
+    }
+
+    @Test
+    public void basicUnderlineButtonToggleTest() {
+        String pressedAttribute = "aria-pressed";
+        basicButtonToggleUnderline.click();
+        basicButtonToggleUnderline.has().attr(pressedAttribute, "true");
+        basicButtonToggleUnderline.click();
+        basicButtonToggleUnderline.has().attr(pressedAttribute, "false");
+    }
+```
+
```

Comment:
![image](https://user-images.githubusercontent.com/65406596/87914287-17f4e380-ca79-11ea-8d7c-346b1e325e1f.png)
Please move code example to related element.